.env.sample

```diff
@@ -0,0 +1,5 @@
+# Auth provider, possible values are 'oidc' or 'basic'
```

Comment:
@huy-ta use `.env.example` name for sample env file for the application grid. So I think we should have one sample file.

Reply:
@tuanlc : do you mean that we agree on the name `.env.example` instead of `.env.sample`, right?
This sentence `So I think we should have one sample file.` makes me confused

Reply:
Yes, that it

Reply:
Yes the popular convention is `env.example`.